src/main/java/edu/ie3/datamodel/utils/validation/GridContainerValidationUtils.java

```diff
@@ -343,4 +359,19 @@ private static InvalidGridException getMissingNodeException(AssetInput input) {
             + input
             + " is connected to a node that is not in the set of nodes.");
   }
+
+  /**
+   * Builds an exception, that announces that the nodes of a transformer are located at the wrong
+   * voltage side
+   *
+   * @param input Input model
+   * @return Exception for the wrong voltage side
+   */
+  private static InvalidGridException getWrongVoltageSideException(AssetInput input) {
+    return new InvalidGridException(
+        input.getClass().getSimpleName()
+            + "The nodes of transformer"
+            + input
+            + " are located at the wrong voltage side");
+  }
```

Comment:
Since we only use it once we for now I think we don't need a more generic method here. Also you could add that we expect the node a to have the higher voltage level into the error message.